packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -207,6 +207,13 @@ export class AtomicSearchBox {
     this.activeDescendant = activeDescendant;
   }
 
+  private updateActiveDescendantWithHistory(activeDescendant = '') {
```

Comment:
What does `withHistory` mean here ?

Reply:
What hapepns if we quickly move between 3 different results? Would this still work correctly?

Reply:
`withHistory` means that we are keeping track of the previous element (when we change panels). I can rename to `updateActiveAndPreviousDescendants`, is that better? @olamothe